packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4895,8 +4897,8 @@ class RenderFollowerLayer extends RenderProxyBox {
 
```

Comment:
we can make the three fields here non-nullable by putting their initializors in the constructor